scriptmodules/supplementary/skyscraper.sh

```diff
@@ -658,3 +610,195 @@ function gui_skyscraper() {
         fi
     done
 }
+
+function _gui_cache_skyscraper() {
+    local db_size
+    local cache_folder=$(_cache_folder_skyscraper)
+    local -A help_strings_cache
+
+    iniConfig " = " '"' "$configdir/all/skyscraper.cfg"
+    eval $(_load_config_skyscraper)
+
+    help_strings_cache=(
+        [1]="Toggle whether screenshots are cached locally when scraping.\n\nSkyscraper option: \Zb--noscreenshots\Zn"
+        [2]="Toggle whether covers are cached locally when scraping.\n\nSkyscraper option: \Zb--nocovers\Zn"
+        [3]="Toggle whether wheels are cached locally when scraping.\n\nSkyscraper option: \Zb--nowheels\Zn"
+        [4]="Toggle whether marquees are cached locally when scraping.\n\nSkyscraper option: \Zb--nomarquees\Zn"
+        [5]="Force the refresh of resources in the local cache when scraping.\n\nSkyscraper option: \Zb--refresh\Zn"
+        [P]="Purge \ZbALL\Zn all cached resources."
+        [S]="Purge all cached resources for a chosen system."
+#        [V]="Removes all non-used cached resources for a chosen system (Vacuum)"
```

Comment:
this commented line is needed ?

Reply:
It's an option that will only work in the 3.x version. 
I removed it when I realized the users might no upgrade to it right away, I commented it thinking I'll enable it on a future update. The corresponding action is left untouched.
If you prefer to remove it, it's fine.